pkg/controller/cincinnati/new.go

```diff
@@ -496,3 +564,7 @@ func checksumMap(m map[string]string) (string, error) {
 
 	return fmt.Sprintf("%x", hash.Sum(nil)), nil
 }
+
+func (k *kubeResources) UseInitContainer(instance *cv1alpha1.Cincinnati) bool {
```

Comment:
I don't think we need this if `newGraphDataInitContainer` is updated to return `nil` if the image isn't specified.